tests/pos/t0301.scala

```diff
@@ -6,7 +6,7 @@ case class Var() extends Expr
 object Analyzer {
   def substitution(expr: Expr, cls: (Var,Var)): Expr =
     expr match {
-      case cls._2 => cls._1  // source of the error
+      case e if e == cls._2 => cls._1  // source of the error
```

Comment:
Since `_2` is now added thought implicit conversion, the pattern used in this case is no longer stable. ([see commit message](https://github.com/lampepfl/dotty/pull/2199/commits/d06e3203c843e24578368a996287bea61b466a88))